ENGAGEHF.xcodeproj/project.pbxproj

```diff
@@ -273,6 +285,53 @@
 			path = Helper;
 			sourceTree = "<group>";
 		};
+		4DB025B52BBDE8EC002D2545 /* Dashboard */ = {
+			isa = PBXGroup;
+			children = (
+				4DB025B62BBDE903002D2545 /* DashboardView.swift */,
+			);
+			path = Dashboard;
+			sourceTree = "<group>";
+		};
+		4DB025B92BBDE9CB002D2545 /* HeartHealth */ = {
+			isa = PBXGroup;
+			children = (
+				4DB025BC2BBDE9F7002D2545 /* HeartHealthView.swift */,
+			);
+			path = HeartHealth;
+			sourceTree = "<group>";
+		};
+		4DB025BA2BBDE9DA002D2545 /* Medications */ = {
+			isa = PBXGroup;
+			children = (
+				4DB025C02BBDEA0B002D2545 /* MedicationsView.swift */,
+			);
+			path = Medications;
+			sourceTree = "<group>";
+		};
+		4DB025BB2BBDE9E2002D2545 /* Education */ = {
+			isa = PBXGroup;
+			children = (
+				4DB025BE2BBDEA03002D2545 /* EducationView.swift */,
+			);
+			path = Education;
+			sourceTree = "<group>";
+		};
+		4DB025C32BBE219B002D2545 /* Functions */ = {
+			isa = PBXGroup;
+			children = (
+				4DB025C52BBE21EF002D2545 /* index.js */,
+			);
+			path = Functions;
+			sourceTree = "<group>";
+		};
+		4DB025C72BBE24B9002D2545 /* Firebase */ = {
+			isa = PBXGroup;
+			children = (
+			);
+			path = Firebase;
+			sourceTree = "<group>";
+		};
```

Comment:
There is an non-existent Firebase folder in here that is not used in the Xcode project, please remove this from he Xcode sidebar:
<img width="262" alt="Screenshot 2024-04-04 at 12 06 32 AM" src="https://github.com/StanfordBDHG/ENGAGE-HF/assets/28656495/25fdf411-bd9b-4a8e-963f-a4bfc9a5607a">